src/researchhub_case/views/author_claim_case_view.py

```diff
@@ -122,12 +124,24 @@ def _get_author_claim_cases_for_mods(self, request):
 
             target_case_set = AuthorClaimCase.objects.filter(
                 status__in=case_query_status
+            ).order_by('-updated_date')
+            page_num = int(request.query_params.get('page')) or 1
+            paginator = Paginator(target_case_set, self.pagination_size)
+            pagination = paginator.page(page_num)
+
+            return Response(
+                data={
+                    'count': paginator.count,
+                    'has_more': pagination.has_next(),
+                    'page': page_num,
+                    'result': AuthorClaimCaseSerializer(
+                        pagination.object_list,
+                        many=True
+                    ).data,
+                },
+                status=200
```

Comment:
Was there a reason to not use the default pagination method?

Ex:
```
            page = self.paginate_queryset(all_documents)
            serializer = self.serializer_class(page)
            serializer_data = serializer.data
            return self.get_paginated_response(serializer_data)
```

Reply:
ran